scripts/package

```diff
@@ -15,12 +15,22 @@ cp ../bin/pipeline ./pipeline/
 
 TAG=${TAG:-${VERSION}${SUFFIX}}
 REPO=${REPO:-rancher}
+#Tag for dependent images
+PIPELINE_UI_VERSION=1.0.1
+PIPELINE_UI_TAG=v${PIPELINE_UI_VERSION}
```

Comment:
this seems redundant. can't we just use `v${PIPELINE_UI_VERSION}` for the tag section in the scripts below?

(same comment for the other `TAG` variables)